code-challenges.test.js

```diff
@@ -15,46 +15,146 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// describe(fibonacciMe, ()=> {
+//     it('Is a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence', ()=> {
+//     expect(fibonacciMe(6)).toEqual([1, 1, 2, 3, 5, 8])
+//     })
+//     })
+
+
 // Example input: 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 
-
 // b) Create the function that makes the test pass.
+//Psuedo Coding
+//First I need to create a function 
+// I need the function to take in a number and return it as an array 
+//I need the array to spit out the fibonacci sequence starting from 2 with the end point being the number I input.
+// I need more than one function to factor this out, this has escalated to a class. 
+
+const fibonacciMe = function(num) {
+    var fib = [];
+    num < 2 ? fib.push(num) : fib.push(fibonacciMe(num-1) + fibonacciMe(num-2));
+    return fib;
+};
+
 
+// var num1 = 1
+// const fibonacciMe = (input) =>{
+//     let newArr = []  
+//     for (let index = 0; index < input.length; index++) {
+//     return newArr((num1 -1) + (num1 -2))
+//     }
+//     return newArr.push(fibonacciMe)
+
+// }
```

Comment:
Okay both of these solutions are close! 
- What issues did you run into? 
- What responses did you get? 
- Did you get the tests to pass? 

This is all necessary information if others are going to be able to help you.